edge/pkg/edged/edged.go

```diff
@@ -259,7 +267,7 @@ func MakeKubeClientBridge(kubeletDeps *kubelet.Dependencies) {
 	kubeletDeps.HeartbeatClient = client
 }
 
-func (e *edged) handlePod(op string, content []byte, podCfg *config.PodConfig) (err error) {
+func (e *edged) handlePod(op string, content []byte, updatesChan chan<- interface{}) (err error) {
```

Comment:
why modify edged.go like this? this is not related to k8s vendor bumping?

Reply:
> why modify edged.go like this? this is not related to k8s vendor bumping?

refer： #4470